### DISCUSSION TITLE
Directus docker image not passing (DCT)

### DISCUSSION THREAD

**Discussion by Saranpjs727** [directus/directus#17250] [Archive]
We are using the directus image as a base image in the docker file.
For security reasons, we need to verify 'Docker Content Trust' using notary to make sure the image is signed before pulling.
But directus image is not signed. 
Why is there no signature on the image? Is it safer to bypass the "Docker Content Trust" check?

**Comment by rijkvanzanten** [comment]
> Why is there no signature on the image?

Cause we haven't implemented it in our CI/CD pipeline. We're currently revising that build pipeline, so we might be able to include that in the future.
